hipamd/hip-config-amd.cmake

```diff
@@ -148,14 +148,14 @@ execute_process(
   RESULT_VARIABLE CLANGRT_BUILTINS_FETCH_EXIT_CODE)
 
 if( CLANGRT_Error )
-  message( STATUS "${HIP_CXX_COMPILER}: CLANGRT compiler options not supported.")
+  message(DEBUG "${HIP_CXX_COMPILER}: CLANGRT compiler options not supported.")
```

Comment:
This also makes a statement about the host compiler so I can see some usefulness from a user perspective. I would put it as a warning though.